applications/nrf_desktop/src/modules/hids.c

```diff
@@ -28,31 +28,26 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_HIDS_LOG_LEVEL);
 
-
 #define BASE_USB_HID_SPEC_VERSION   0x0101
 
-
-BT_GATT_HIDS_DEF(hids_obj,
-		 IF_ENABLED(CONFIG_DESKTOP_HID_REPORT_MOUSE_SUPPORT,
-			    (REPORT_SIZE_MOUSE,))
-		 IF_ENABLED(CONFIG_DESKTOP_HID_REPORT_KEYBOARD_SUPPORT,
-			    (REPORT_SIZE_KEYBOARD_KEYS,
-			     REPORT_SIZE_KEYBOARD_LEDS,))
-		 IF_ENABLED(CONFIG_DESKTOP_HID_REPORT_SYSTEM_CTRL_SUPPORT,
-			    (REPORT_SIZE_SYSTEM_CTRL,))
-		 IF_ENABLED(CONFIG_DESKTOP_HID_REPORT_CONSUMER_CTRL_SUPPORT,
-			    (REPORT_SIZE_CONSUMER_CTRL,))
-		 IF_ENABLED(
-			 CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE,
-			 (REPORT_SIZE_USER_CONFIG, /* HID feature report. */
-			  IF_ENABLED(CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT,
-				     /* HID output report. */
-				     (REPORT_SIZE_USER_CONFIG,))))
-
-		 0 /* Appease macro with a dummy zero */
+BT_HIDS_DEF(hids_obj,
```

Comment:
There are plenty of ways code can be beautified; since beauty is in the eye of a beholder it is subjective :)
Even if it is tempting to change style in code, please don't do it unless necessary for original purpose, especially for changes that are +2,163 −2,206 big.

Years ago I have heard something similar. At that time I thought that it's gibberish, my modification had made that code more readable and so are perfectly excused.
Now, after reviewing n-th file where I have to carefully read the change instead of spending few seconds on each section prove how silly I was.

Reply:
It's fine